### PR TITLE
Bump versions for development packages

### PR DESCRIPTION
**Pull request by jamiehannaford**
Addresses #581. FAO @ycombinator @seeafish

### CODE REVIEW

**Comment by jamiehannaford** [comment]
The 5.3 build should fail (composer installs dev packages by default now). I'm pretty sure this will only affect users who are cloning the git repo and running `composer install`. For package users running `composer require rackspace/php-opencloud`, dev packages will not be installed and 5.3 _should_ work.

**Comment by seeafish** [comment]
@jamiehannaford Yes, running `php composer.phar install --no-dev` works fine as well. It is only for development that this is a problem.
However, I am unsure if that would mean accidentally writing code that may not work on php 5.3, since all development will suddenly happen on 5.4 and above.

**Comment by jamiehannaford** [comment]
@seeafish Me and @ycombinator have had a quick chat about this and we think bumping the overall minimum requirement to 5.4 since 5.3 is fully into EOL and no longer receives security updates. 5.3 users can still use the SDK, but will have to specify `1.12` or lower as their stable version.

File: doc/using-php-5.3.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Using the SDK with PHP v5.3
```
Comment (ycombinator) [inline doc/using-php-5.3.rst]: Great idea to create this document. Do you think we should link to it from http://docs.php-opencloud.com/en/latest/#installation?

**Comment by ycombinator** [comment]
Could you also bump up [`OpenCloud\Version::VERSION`](https://github.com/rackspace/php-opencloud/blob/working/lib/OpenCloud/Version.php#L30) as part of this PR, please?

**Comment by ycombinator** [comment]
LGTM. Merging.

**Comment by seeafish** [comment]
:+1: :)

**Pull request merged**